CONTRIBUTING.md

```diff
@@ -1,57 +1,108 @@
+* [Prerequisites](#prerequisites)
+* [Getting started](#getting-started)
+* [Adding a new component](#adding-a-new-component)
+* [How we review Backpack contributions](#how-we-review-backpack-contributions)
+* [How to](#how-to)
 
-**All new components should be written in Swift.**
-
-Please follow the [New York Times Objective-C style-guide](https://github.com/NYTimes/objective-c-style-guide) when writing Objective-C. Follow other conventions and patterns established in the source code already when the style-guide cannot help you. The goal is that the codebase should look like it was written by a single author.
+## Prerequisites
 
-Wrap all Objective-C in `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` blocks. Make sure to annotate any types that are nullable correctly.
+### Licence
+By contributing your code, you agree to license your contribution under the terms of the [APLv2](./LICENSE).
 
-## How we review Backpack contributions
-
-Please see the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md).
+All files are released with the Apache 2.0 licence.
 
-## Environment
+### Environment
 
 We use both Ruby and Node in this project. To manage these language runtimes we recommend using [`rbenv`][0] and [`nvm`][1] respectively. You should ensure you have these installed or some other means of handling Ruby and Node versions. The required ruby version is specified in `.ruby-version` and the Node version is in `.nvmrc`.
 
 With `rbenv` use `rbenv install` to install the project's version of Ruby. For `nvm` use `nvm use`. Also ensure you install `bundler` for Ruby with `gem install bundler`.
 
 Use the most recent stable version of Xcode, however the project should work with old versions of the same major.
 
-## Setup
+### Code style
+
+**All new components should be written in Swift.**
+
+Please follow the [New York Times Objective-C style-guide](https://github.com/NYTimes/objective-c-style-guide) when writing Objective-C. Follow other conventions and patterns established in the source code already when the style-guide cannot help you. The goal is that the codebase should look like it was written by a single author.
+
+Wrap all Objective-C in `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` blocks. Make sure to annotate any types that are nullable correctly.
 
+## Getting started
 Given that you have a compatible environment as stated above you can now setup the project.
 
 - `bundle install` to install ruby dependencies
 - `npm install` to install npm dependencies
-- If you work for Skyscanner, [configure relative](#relative-font) (optional).
 - `(cd Example && bundle exec pod install)` To setup the example project.
 - `open Example/Backpack.xcworkspace` to open the example project
 
-## Relative Font
+## Adding a new component
+If you want to add a new component, we will need the following:
 
-> Skyscanner employees only
+- Design (Figma file)
+- UIKit and SwiftUI component
```

Comment:
is it fair to ask teams to contribute two versions when they only need one?

Reply:
changed to OR